webapp/src/components/rest/AuthRestConnector.js

```diff
@@ -0,0 +1,12 @@
+import RestConnector from '@/components/rest/RestConnector'
+
+class AuthRestConnector extends RestConnector {
+
+    constructor(token){
+        super();
+        this.token = token;
+    }
+
```

Comment:
Deze klas heeft nog geen functionaliteit. Komt dit nog? Of is deze klas niet meer nodig?